src/components/Toast/toast.st.css

```diff
@@ -0,0 +1,117 @@
+.toast-region {
+  position: fixed;
+  top: 16px;
+  right: 16px;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+}
+
+@keyframes slide-out {
+  0% {
+    transform: translateX(0);
+    opacity: 1;
+  }
+
+  100% {
+    transform: translateX(100%);
+    opacity: 0;
+  }
+}
+
+@keyframes slide-in {
+  0% {
+    transform: translateX(100%);
+    opacity: 0;
+  }
+
+  100% {
+    transform: translateX(0);
+    opacity: 1;
+  }
+}
+
+@keyframes fade-in {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+.root {
+  -st-states: priority(enum(info, success, warning, error));
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 16px;
+  background: slateblue;
+  color: white;
+  padding: 12px 16px;
```

Comment:
As a general rule we want to keep styles in the api files to a point that the component 'works' and we can see it and that is about it, some general alignment stuff is fine too, the rest can be defined in the default theme.

Reply:
Okay, makes sense. Will move the appropriate styles to the theme.

Reply:
Done.